tests/e2e/execute-tests-cloud-function.sh

```diff
@@ -13,9 +13,14 @@
 #     See the License for the specific language governing permissions and
 #     limitations under the License.
 
-export START_LOAD_GENERATION=$(date -u +%s%3N)
+START_LOAD_GENERATION=$(date -u +%s%3N)
```

Comment:
Same as "readonly", why is this split? It gives lots of unnecessary volume

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2155